src/pybind/mgr/dashboard/frontend/src/styles.scss

```diff
@@ -344,6 +344,13 @@ uib-accordion .panel-title,
 .ceph-icon {
   background: url('assets/Ceph_Logo_Stacked_RGB_120411_fa_228x228.png');
 }
+.toast-message .ceph-icon {
```

Comment:
This doesn't work for all icons in use.

Reply:
Currently there are only 2 icons as far as i can see. The Prometheus icon is already white, so only the Ceph icon must be modified.

Reply:
My initial idea was NOT to display all icons using white color (maybe other upcoming icons would better match with green and the orther notification background colors). But maybe it is better to display ALL icons using white for better visibility.

Reply:
Fixed

Reply:
Hm... you are right, a white dot isn't any better (happens with your current change). So please revert the change or use a different prometheus icon with no white infill. Like https://github.com/prometheus/docs/blob/master/static/prometheus_logo_orange_circle.svg or https://github.com/prometheus/docs/blob/master/static/prometheus_logo_orange_invert.svg

Reply:
I've replaced the Prometheus PNG logo with the SVG counterpart which renders the transparent areas correctly.